proyectos/objetivo-1.md

```diff
@@ -38,7 +38,7 @@
 | https://github.com/delightfulagony/oishi/pull/11 | v0.1.0 | ✓ |
 | https://github.com/pablo1mc315/iv_pablomc/pull/2 | v0.0.1 | ✓  |
 | <!-- Enlace de antoniojesuus --> | | |
-| <!-- Enlace de ottoeprz --> | | |
+| https://github.com/ottoeprz/glowing-engine/pull/2 | v0.0.1 | |
```

Comment:
Ya tienes esa versión [aquí](https://github.com/JJ/IV-22-23/blob/master/proyectos/objetivo-0.md). Por favor, haced un pequeño esfuerzo en entender lo que significa una versión en el contecto de desarrollo ágil y mirad las versiones que ya habéis puesto en el pasado para cambiarlas.